.reek.yml

```diff
@@ -5,3 +5,17 @@ detectors:
   UtilityFunction:
     exclude:
       - 'Inquisition::CLI#execute'
+
+  TooManyInstanceVariables:
```

Comment:
I suggest rolling back these changes (and remove `UtilityFunction`) because the main our goal is to fix issues and only if it does not make sense - disable such checks